stdlib/2and3/logging/__init__.pyi

```diff
@@ -79,6 +80,7 @@ class Logger:
         def exception(self,
                       msg: Text, *args: Any, exc_info: _ExcInfoType = ...,
                       extra: Dict[str, Any] = ..., **kwargs: Any) -> None: ...
+        def isEnabledFor(self, lvl: int) -> Union[int, bool]: ...
```

Comment:
You're right, the only "int" ever returned is 0 which for all intents and purposes is equivalent to False unless somebody does "is False" checks. Will simplify further.